spec/lib/gitlab/github_import/client_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Gitlab::GithubImport::Client do
+  it 'test' do
+    client = Gitlab::GithubImport::Client.new(nil)
+    authorize_url = "https://github.com/login/oauth/authorize?" +
+                    "response_type=code&" +
+                    "client_id=YOUR_APP_ID&" +
+                    "redirect_uri=https%3A%2F%2Fapp.com&" +
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.